scripts/build/index.js

```diff
@@ -23,7 +23,7 @@ const ddocsBuildPath = path.resolve(buildPath, 'ddocs');
 
 const getCouchConfig = () => {
```

Comment:
Is it just me or is this function not used anywhere any more?

Reply:
Heh, you're right!